crates/ark-contracts/arkchain/src/lib.cairo

```diff
@@ -10,4 +10,3 @@ mod crypto {
 }
 
 mod orderbook;
-mod tests_lib;
```

Comment:
why did you remove tests_lib ? it should be included, actually i move the setup function here so we can reuse the setup, can you do that in you test also ? in the same file

Reply:
I just moved your setup function into the same directory where the tests are located